app/controllers/admin/variant_overrides_controller.rb

```diff
@@ -30,6 +29,22 @@ def bulk_update
       end
     end
 
+    def bulk_reset
+      collection_hash = Hash[params[:variant_overrides].each_with_index.map { |vo, i| [i, vo] }]
```

Comment:
Probably best to extract the shared code out and place it in a before_filter which is called for each method. Something like:

`before_filter :load_collection, only: [:bulk_update, :bulk_reset]`

then below (in the private block):

    def load_collection
      collection_hash = Hash[params[:variant_overrides].each_with_index.map { |vo, i| [i, vo] }]
      vo_set = VariantOverrideSet.new @variant_overrides, collection_attributes: collection_hash
      # Ensure we're authorised to update all variant overrides
      vo_set.collection.each { |vo| authorize! :update, vo }
    end